### PR TITLE
Feature/53 setup rest

### LINKED ISSUE #53: Set up REST

### PR DESCRIPTION
**Pull request by JustMilan**
## Changes
Added endpoints for:
- registering a customer
- getting reviews from a customer
- getting orders from a customer
- getting deliveries from a customer
- getting address from customer by customer id
- getting address from customer by order id

## Checklist
- [x] Create rest endpoints
- [x] Add tests
- [x] Format all modified files
- [x] Check for missing newlines
- [x] optimize imports

## When merging, this pull request 
closes #53

### CODE REVIEW

File: review/src/main/java/nl/softwarestrijders/waiter/review/core/application/CommandHandler.java [hunk 1]
```diff
@@ -18,55 +18,54 @@
...
+			throw new InvalidOperationException();
+		}
 
-        this.repository.delete(review);
-        this.eventPublisher.publish(new ReviewDeletedEvent(review.getId(), review.getType()));
-    }
+		this.repository.delete(review);
+		this.eventPublisher.publish(new ReviewDeletedEvent(review.getId(), review.getType()));
+	}
 }
```
Comment (xandervedder) [inline review/src/main/java/nl/softwarestrijders/waiter/review/core/application/CommandHandler.java:71]: :thinking:

File: customer/src/main/java/nl/softwarestrijders/waiter/customer/infrastructure/driver/web/CustomerController.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	public List<UUID> getOrdersFromCustomer(@PathVariable UUID customerId) {
+		return this.queryHandler.handle(new GetOrdersFromCustomer(customerId));
+	}
+
+	@GetMapping("/deliveries/{customerId}")
+	public List<UUID> getDeliveriesFromCustomer(@PathVariable UUID customerId) {
+		return this.queryHandler.handle(new GetDeliveriesFromCustomer(customerId));
+	}
+
+	@GetMapping("/order/{orderId}/retrieve-address")
```
Comment (xandervedder) [inline customer/src/main/java/nl/softwarestrijders/waiter/customer/infrastructure/driver/web/CustomerController.java:61]:
```suggestion
        // TODO: change URL name
	@GetMapping("/order/{orderId}/retrieve-address")
```

**Review by xandervedder — APPROVED** [review]

File: customer/postman/BEP03.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+						"type": "text/javascript",
+						"exec": [
+							""
+						]
+					}
+				}
+			]
+		}
+	]
+}
```
Comment (xandervedder) [inline customer/postman/BEP03.postman_collection.json:230]:
```suggestion
}

```

File: customer/src/main/java/nl/softwarestrijders/waiter/customer/core/application/exception/OrderNotFoundException.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package nl.softwarestrijders.waiter.customer.core.application.exception;
+
+import java.util.UUID;
+
+public class OrderNotFoundException extends RuntimeException {
+	public OrderNotFoundException(UUID id) {
+		super(String.format("Order with id: %s has not been found!", id.toString()));
+	}
```
Comment (xandervedder) [inline customer/src/main/java/nl/softwarestrijders/waiter/customer/core/application/exception/OrderNotFoundException.java:8]: Unused code

**Review by arjennorbart — APPROVED** [review]

**Pull request merged**